### PR TITLE
fix: Title missing in thread list for threads starting with attachment in E2EE rooms

### PR DESCRIPTION
**Pull request by hugocostadev**
## Proposed changes (including videos or screenshots)

The thread title is not showing on the threads list if the thread starts with an image or a file

Before:
image

Now:
image

## Steps to test or reproduce

1. Open an E2EE channel
2. Send an image or file to the channel
3. Hover the image → Start the thread
4. Send a message inside the thread
5. On the top of the channel, click on “Threads” to see the threads list

## Further comments

https://rocketchat.atlassian.net/browse/E2EE2-51

### CODE REVIEW

File: .changeset/moody-phones-admire.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@rocket.chat/meteor": minor
+---
+
+Fixed thread list contextual bar when the first message is an attachment
```
Comment (aleksandernsilva) [inline .changeset/moody-phones-admire.md:5]: https://developer.rocket.chat/docs/development-workflow
> Verb: Start with a verb in the third-person singular present tense.

**Review by aleksandernsilva — CHANGES_REQUESTED** [review]
Hey @hugocostadev , I tested the branch and everything seem to be working fine 👌 
Just the minor adjustment to the changeset. 

Btw, I belive we could improve this PR's title to be a bit more descriptive (e.g `fix: Title missing in thread list for threads starting with attachment in E2EE rooms`)

_I know my title suggestion is a bit long but it's just an example_ 😉

File: .changeset/moody-phones-admire.md (hunk 1 above)
Comment (hugocostadev) [reply .changeset/moody-phones-admire.md]: much better! thanks man

File: .changeset/moody-phones-admire.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"@rocket.chat/meteor": minor
```
Comment (aleksandernsilva) [inline .changeset/moody-phones-admire.md:2]:
```suggestion
"@rocket.chat/meteor": patch
```

**Review by aleksandernsilva — CHANGES_REQUESTED** [review]

**Review by aleksandernsilva — APPROVED** [review]

**Pull request merged**